web-previewer/src/components/useWalletState.js

```diff
@@ -12,27 +15,52 @@ export default function useWalletState() {
         tokens,
 
         async connect() {
-            if (!window.ethereum) {
-                alert(
-                    'MetaMask not detected. Please try again from a MetaMask enabled browser.'
-                )
+            const [_address] = await window.ethereum.request({method: "eth_requestAccounts"})
+            address.value = _address
+            await this.fetchOwnersTokens();
+        },
 
```

Comment:
Since I haven't used `ethers.js` before: Which providers / wallets are supported out-of-the-box? 

I'm guessing WalletConnect is still a separate integration, correct?